src/utils.js

```diff
@@ -592,3 +592,39 @@ Calculate the available width for a chart.
 nv.utils.availableWidth = function(width, container, margin) {
     return nv.utils.sanitizeWidth(width, container) - margin.left - margin.right;
 }
+
+/*
+Clear any rendered chart components and display a chart's 'noData' message
+*/
+nv.utils.noData = function(chart, container) {
+    var opt = chart.options(),
+        margin = opt.margin(),
+        noData = opt.noData(),
```

Comment:
Lets do noData = opt.noData() || "No Data Available", and set the default of noData to null in the rest of the models... that way the default string is only defined in one place.